<a href="https://colab.research.google.com/github/FelipeGonzalezGarrido/predicciones-de-ventas/blob/main/Proyecto_2_base.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
from google.colab import drive
drive.mount ("/content/drive")

Mounted at /content/drive


In [2]:
import pandas as pd

In [3]:
filename = "/content/drive/MyDrive/Data CodingDojo/car_price.csv"
df = pd.read_csv(filename)
df.head()

Unnamed: 0                                car_name car_prices_in_rupee  \
0           0  Jeep Compass 2.0 Longitude Option BSIV          10.03 Lakh   
1           1            Renault Duster RXZ Turbo CVT          12.83 Lakh   
2           2                      Toyota Camry 2.5 G          16.40 Lakh   
3           3                       Honda Jazz VX CVT           7.77 Lakh   
4           4        Volkswagen Polo 1.2 MPI Highline           5.15 Lakh   

   kms_driven fuel_type transmission  ownership  manufacture   engine    Seats  
0  86,226 kms    Diesel       Manual  1st Owner         2017  1956 cc  5 Seats  
1  13,248 kms    Petrol    Automatic  1st Owner         2021  1330 cc  5 Seats  
2  60,343 kms    Petrol    Automatic  1st Owner         2016  2494 cc  5 Seats  
3  26,696 kms    Petrol    Automatic  1st Owner         2018  1199 cc  5 Seats  
4  69,414 kms    Petrol       Manual  1st Owner         2016  1199 cc  5 Seats

In [5]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 5512 entries, 0 to 5511
Data columns (total 10 columns):
 #   Column               Non-Null Count  Dtype 
---  ------               --------------  ----- 
 0   Unnamed: 0           5512 non-null   int64 
 1   car_name             5512 non-null   object
 2   car_prices_in_rupee  5512 non-null   object
 3   kms_driven           5512 non-null   object
 4   fuel_type            5512 non-null   object
 5   transmission         5512 non-null   object
 6   ownership            5512 non-null   object
 7   manufacture          5512 non-null   int64 
 8   engine               5512 non-null   object
 9   Seats                5512 non-null   object
dtypes: int64(2), object(8)
memory usage: 430.8+ KB


Primera opción: conjunto de datos 1

1) Fuente de datos

[Kaggle](https://www.kaggle.com/datasets/milanvaddoriya/old-car-price-prediction)

2) Breve descripción de datos

Contiene informacion sobre autos usados como:
- Kilometraje
- Precio
- Añio de manufactura
- Asientos
- Tipo de combustible

3) ¿Cuál es el objetivo?

El objetivo es predecir el precio de un auto usado segun sus caracteristicas.

4) ¿Este es un problema de clasificación o regresión?

Como se busca identificar o predecir un precio, el cual es un valor continuo. Se trata de un problema de regresion.

5) ¿Cuántas características hay?

El conjunto de datos posee 8 caracteristicas.

6) ¿Cuántas filas de datos hay?

Tiene 5512 filas o registros.

7) ¿Qué retos, si es que hay alguno, prevén en la limpieza, exploración o modelización con este conjunto de datos?

El conjunto cuenta con columnas como el precio, el cual sera procesado para un mejor entendimiento a dolar.
tambien regularizar los valores de kilometros recorridos y la codificacion en general del conjunto.